tests/unit/test_users.py

```diff
@@ -272,3 +278,16 @@ def test_str(self):
     def test_is_free(self):
         """Show that user can check if the plan is free."""
         assert self.instance.is_free() is False
+
+
+class TestAuthenticatedUserCompatibility_2_12(helper.UnitHelper):
+
+    """Test methods on AuthenticatedUser from Github Enterprise 2.12."""
+
```

Comment:
The flake8 tester is saying there are too many blank lines here. Just take one out and that should appease the CI.